internal/service/debug.go

```diff
@@ -0,0 +1,13 @@
+package service
+
+import (
+	"github.com/etf1/kafka-mongo-watcher/internal/debug"
+)
+
+func (container *Container) GetDebugger() *debug.Debugger {
+	if container.debugger == nil {
```

Comment:
Maybe we can just instantiate the debugger if `container.Cfg.HttpServer.DebugEnabled == true` AND return `nil` if it's not the case.